src/libPMacc/include/memory/buffers/MultiGridBuffer.hpp

```diff
@@ -174,11 +174,11 @@ class MultiGridBuffer
     }
 
     /**
-     * Starts sync data from own device buffer to neigbhor device buffer.
+     * Starts sync data from own device buffer to neighboring device buffer.
      *
-     * Asynchronously starts syncronization data from internal DeviceBuffer using added
+     * Asynchronously starts synchronization of data from internal DeviceBuffer using added
      * Exchange buffers.
-     * This operation runs sequential to other code but intern asyncron
+     * This operation runs sequential to other code but uses asynchronous operations internally.
```

Comment:
I think `sequential` describs the verb `runs`? Then, it has to be an adverb: `sequentially`?